pipelines/batch/src/main/java/com/google/fhir/analytics/JdbcResourceWriter.java

```diff
@@ -85,65 +97,92 @@ static void createTables(FhirEtlOptions options)
     log.info(
         String.format(
             "Connecting to DB url %s with user %s.", dbConfig.jdbcUrl(), dbConfig.dbUser()));
-    if (options.getUseSingleSinkTable()) {
```

Comment:
I am assuming that we are removing the support for this feature where it writes everything to one single table.

Reply:
Ok